service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/request/EhrExtractRequestHandler.java

```diff
@@ -126,14 +126,31 @@ public void handleContinue(String conversationId, String payload) {
             ehrExtractStatusService.updateEhrExtractStatusContinue(conversationId)
                 .ifPresent(ehrExtractStatus -> {
                     var documents = ehrExtractStatus.getGpcAccessDocument().getDocuments();
+                    if (ehrExtractStatus.getGpcAccessStructured() != null) {
```

Comment:
I would put that into seperate method for readability - i.e. looks like this methods logs something when a codition is met